analytics/common/build.sh

```diff
@@ -2,4 +2,15 @@
 
 DIR=$(dirname $(readlink -f "$0"))
 
+download_image(){
+	URL=https://raw.githubusercontent.com/OpenVisualCloud/Dockerfiles/v22.3/Xeon/ubuntu-20.04/analytics/gst/Dockerfile
```

Comment:
add a if not exist then download logic. Otherwise, everything you do make, it will cause a fresh new build. Also add the Dockerfile to .gitignore.

Reply:
has add these